R/brdg_summary.R

```diff
@@ -575,45 +575,49 @@ pier_summary_missing_row <- function(cards,
         .data$stat_name %in% c("N", "n", "p", "N_unweighted", "n_unweighted", "p_unweighted")
       )
 
+    # if not tabulation of the by variable provided, just give return the by levels
```

Comment:
Minor improvements to code comment
```suggestion
    # if no tabulation of the 'by' variable provided, just return the 'by' levels
```

Reply:
thank you! this is more clear :)